BranchMetrics/xamarin-branch-deep-linking-attribution
# [SDK-1734] Release 8.1.1

## Reference
SDK-1734 -- [Xamarin] Release 8.1.1

## Summary
Updated the iOS binary to Branch iOS SDK v1.45.1 to fix SKAN error.

## Motivation
Xamarin was recently updated to the iOS SDK v1.45.0 which introduced a bug with SKAN. In order to correct. it, Xamarin needs to be updated to the latest, fixed version of the iOS SDK, 1.45.1.

## Type Of Change


## Testing Instructions

Check out the branch, package the Xamarin SDK, and run in an an iOS test app to see the iOS and Xamarin SDK versions have been updated to 1.45.1 and 8.1.1 respectively.
